v1.0.4

sean-e-dietrich opened the pull request.

Commit:
Fixed Readme.md file badge

Commit:
Rewrite of command to not use the new command but to take advantage of existing data.

Commit:
Updated file to start using conatinered methods and data instead of using new classes

Commit:
CI Integration

Added CircleCI build script and added terminus testing script

Commit:
CI Integration

Added test scripts as well as made some code sniffer modifications.

Commit:
Automated Testing

Integration with BATS (Bash Automated Testing System) and wrote 2
initial tests.

Commit:
Fixed typo in bash checking

uberhacker reviewed (commented):
Hey @sean-e-dietrich: This looks pretty good.  There are a few indentation and spacing issues but the code itself looks fine.  What does `composer test` report?  Any suggestions to fix the indentation?  Another thing you might consider is instead of `'{site}: {env} creating backup',`, you could do any of the following:
```
'{site}: {env} backup started',
'{site}: creating {env} backup',
'{site}: creating backup of the  {env} environment',
```
Of course, if you change the message here, you will also need to change in your test cases as well.  The tests look great.  Don't see any problems there.

Commit:
Automated Testing

Fixed testing docs that appear to be working now just seems as though they may be timing out.

sean-e-dietrich commented:
Merging as of now as it looks like the CI builds are timing out due to BATS
